h5p-bildetema/src/components/CollectionsController/CollectionPage/CollectionPage.tsx

```diff
@@ -24,30 +28,45 @@ const CollectionPage = ({
           <BookmarkIcon />
         </div>
         {/* TODO: add translation */}
-        <p>Klikk på bokmerket for ord du vil legge til i denne samlingen.</p>
-        <Button
-          variant="default"
-          role="link"
-          aria-label="button-link to front page to add more words"
-          onClick={() => navigate("/")}
-        >
-          {/* TODO: add translation */}
-          Oppdag ord
-        </Button>
+        <p>Denne samlingen er tom.</p>
+        <p>{description}</p>
+        <div className={styles.navButtons}>
+          <Button
+            variant="default"
+            role="link"
+            aria-label="button-link to front page to add more words"
```

Comment:
Using `aria-label `here will overwrite the actual text within the button. So, "Gå til søk" will never be read by a screen reader. The `aria-label` can be removed as it is not needed.

Reply:
removed aria-label